SPECS/calamares/calamares.spec

```diff
@@ -231,6 +235,12 @@ install -p -m 644 %{SOURCE53} %{buildroot}%{_sysconfdir}/calamares/mariner-eula
 %{_libdir}/cmake/Calamares/
 
 %changelog
+* Tue May 23 2023 Pawel Winogrodzki <pawelwi@microsoft.com> - 3.2.11-42
+- Updating BRs after fixing "kf5" and "Qt5" builds.
+
+* Fri May 19 2023 Pawel Winogrodzki <pawelwi@microsoft.com> - 3.2.11-41
```

Comment:
I think you could merge these two changelog entries together, since they're part of the same PR